neptunecontrib/viz/parallel_coordinates_plot.py

```diff
@@ -0,0 +1,129 @@
+#
+# Copyright (c) 2020, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+
+import hiplot as hip
+import neptune
+
+
+def make_parallel_coordinates_plot(columns=None,
+                                   html_file_path=None,
+                                   id=None,
+                                   state=None,
+                                   owner=None,
+                                   tag=None,
```

Comment:
I ran the following:

```python
import neptune

neptune.init('neptune-ai/credit-default-prediction')
make_parallel_coordinates_plot(tag='features_v1')
```

and got the error:
`
TypeError: data type 'string' not understood
`

I think something is wrong here.

Reply:
Easy fix. I was due to differences in pandas 0.x and 1.0. Now it works with both major version. Thanks